graphics/asymptote/files/patch-Makefile.diff

```diff
@@ -1,38 +1,20 @@
---- Makefile.in.orig
-+++ Makefile.in
-@@ -10,7 +10,7 @@ LIBATOMIC = libatomic_ops-@ATOMICVERSION@
+--- Makefile.in.orig	2022-05-14 12:20:39.000000000 +0700
++++ Makefile.in	2022-08-16 19:46:32.000000000 +0700
+@@ -10,7 +10,7 @@
  GCOPTIONS = @GCOPTIONS@
  GCLIB = @GCLIB@
  GCPPLIB = @GCPPLIB@
 -GCLIBS = $(GCPPLIB) $(GCLIB)
 +GCLIBS = $(GCPPLIB) $(GCLIB) -lgccpp
- LFLAGS = @LDFLAGS@
- LIBS = $(LFLAGS) @PTHREAD_LIBS@ @GLEW@ @LIBS@ $(GCLIBS)
- DOSLIBS = $(subst -lncurses, -ltermcap, $(LIBS)) -lgdi32 -lwinmm -s -static
-#@@ -110,7 +110,7 @@ endif
-# export prefix docdir exampledir mandir infodir INSTALL MAKE DESTDIR TEXI2DVI
-# 
-# asy:	version
-#-	-pyrcc5 GUI/res/icons.qrc -o GUI/icons_rc.py
-#+	$(PYRCC5) GUI/res/icons.qrc -o GUI/icons_rc.py
-# 	if test -n "$(MSDOS)"; then \
-#           $(CXX) $(OPTS) -o $(NAME) $(FILES:=.o) revision.o asy.o $(DOSLIBS); \
-# 	else \
-@@ -137,7 +137,7 @@ version: $(GCLIB) $(FILES:=.o) $(UIFILES:.ui=.py)
- 	echo @set VERSION $(revision) > doc/version.texi
+ LFLAGS = @LDFLAGS@ -LLspCpp -LLspCpp/third_party/uri/src
+ LIBS = $(LFLAGS) @PTHREAD_LIBS@ @GLEW@ @LIBS@ $(GCLIBS) @LSPLIBS@
+ DOSLIBS = $(subst -lncurses, -ltermcap, $(LIBS)) -lwolfssl -lgdi32 -lwinmm -s -static
```

Comment:
Where does `wolfssl` come from? We don't specify any dependency.

Reply:
@mojca That was there in the source, not added by me.

Reply:
this doesn't look resolved yet - correct? I haven't looked carefully  but what @mojca points out is is opportunistic linking in case the wolfssl is installed on a users system. That wouldn't result in a reproducible build; so if it's not needed the library shouldn't be used even if installed.